testing.conf

```diff
@@ -0,0 +1,3 @@
+[global]
```

Comment:
just the example configuration used when pointing Swagger UI at the code that's all.